1/contrib/jenkins/fix-permissions

```diff
@@ -2,5 +2,6 @@
 # Fix permissions on the given directory to allow group read/write of 
 # regular files and execute of directories.
 chgrp -R 0 $1
-chmod -R g+rw $1
+#chmod -R g+rw $1
+find $1 -exec chmod g+rw {} \;
```

Comment:
yeah probably ... i'll wait until the upcoming test completes successfully for the find / chmod before officially pulling that trigger